remoulade/composition.py

```diff
@@ -266,9 +275,10 @@ def get_results(self, *, block=False, timeout=None, raise_on_error=True):
                 timeout = max(0, int((deadline - time.monotonic()) * 1000))
 
             if isinstance(child, group):
-                yield list(child.get_results(block=block, timeout=timeout, raise_on_error=raise_on_error))
+                results = child.get_results(block=block, timeout=timeout, raise_on_error=raise_on_error, forget=forget)
+                yield list(results)
             else:
-                yield child.get_result(block=block, timeout=timeout, raise_on_error=raise_on_error)
+                yield child.get_result(block=block, timeout=timeout, raise_on_error=raise_on_error, forget=forget)
 
     def wait(self, *, timeout=None, raise_on_error=True):
```

Comment:
Add the `forget` param here too. Using `wait` probably means we don't need the result so it could be `True` by default but I'm not sure since it's not consistent.